MoEngagePluginBase/MoEngageHybridToNativeConfig.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MoEngageHybridToNativeConfig.swift
+//  MoEngagePluginBase
+//
+//  Created by UdayKiran Naik on 02/07/24.
+//
+
+import Foundation
+
+public class MoEngageHybridToNativeConfig {
+    public static let sharedInstance = MoEngageHybridToNativeConfig()
+    private var hybridToNativeConfigCache: [String:MoEngageInitConfig] = [:]
+    
+    private init() {
+        
+    }
+    
+    public func initializeInitConfig(appID: String, initConfig: MoEngageInitConfig) {
```

Comment:
From where are we passing  initConfig to this class ?

Reply:
Addressed.

Reply:
It should be populated from the the init json in the pluginInitialized function of MoEngagePluginBridge

Reply:
Addressed.